### PR TITLE
Fix typos in maintenance script

### PR DESCRIPTION
**Pull request by agross**

### CODE REVIEW

File: debian/scripts/maintenance.sh [hunk 1]
```diff
@@ -132,7 +132,7 @@ upgrade() {
   echo 'You are now going to upgrade your js-controller.'
   echo 'As this will change data in /opt/iobroker, make sure you have a backup!'
   echo 'During the upgrade process the container will automatically switch into maintenance mode and stop ioBroker.'
-  echo 'Depending of the restart policy, you container will be stoped/ restarted automatically after the upgrade.'
```
Comment (buanet) [inline debian/scripts/maintenance.sh:135]: :) There is a [DIN 5008](https://de.wikipedia.org/wiki/Schr%C3%A4gstrich#:~:text=8%20Einzelnachweise-,Typografie,Schr%C3%A4gstrich%20kein%20Leerzeichen%20geschrieben%20wird.) for "/". It says no space before and after "/". I would  prefer to change this to "stopped or restarted". Do you agree?

File: debian/scripts/maintenance.sh [hunk 2]
```diff
@@ -149,7 +149,7 @@ upgrade() {
     iobroker upgrade self
     sleep 1
     echo 'Done.'
-    echo 'Container will be stopped/ restarted in 5 seconds...'
```
Comment (buanet) [inline debian/scripts/maintenance.sh:152]: Same here.

**Review by buanet — CHANGES_REQUESTED** [review]

**Comment by buanet** [comment]
Oh, and as there are still some changes made to the maintenance script, I would prefer to change the base branch to "buanet:dev".

File: debian/scripts/maintenance.sh (hunk 1 above)
Comment (agross) [reply debian/scripts/maintenance.sh:135]: Yes, "or" would make it clearer!

File: debian/scripts/maintenance.sh (hunk 2 above)
Comment (agross) [reply debian/scripts/maintenance.sh:152]: Same as above 😉

**Comment by agross** [comment]
I also deduplicated the upgrade code paths.

**Review by buanet — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**